client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -93,11 +91,6 @@ function ChangePaymentMethod( props ) {
 		);
 	}
 
-	const recordFormSubmitEvent = () =>
-		void props.recordTracksEvent( 'calypso_purchases_credit_card_form_submit', {
```

Comment:
I don't see this event anywhere in the new setup... maybe we should migrate it.

Reply:
I think I've actually taken care of this here: https://github.com/Automattic/wp-calypso/pull/48968/commits/690f43c8ba932f9ece1d34c7794cab1b786c226a so you can probably ignore this comment.